Movie/Movie/AppDelegate.swift

```diff
@@ -1,14 +1,37 @@
 // AppDelegate.swift
 // Copyright © RoadMap. All rights reserved.
```

Comment:
```suggestion
// Copyright ©. All rights reserved.
```
Поменяй правила в форматере